thunder/core/jit_ext.py

```diff
@@ -930,6 +931,42 @@ def thunder_function(*args, **kwargs):
     return interpreter_needs_wrap(checkpoint)(wrapped_thunder_function, *args, **kwargs)
 
 
+@register_general_jit_lookaside(torch.Tensor._make_wrapper_subclass)
```

Comment:
Is there a reason for this to be defined as jit_lookaside? It looks simple enough to be just in `thunder/torch/__init__.py` with the `@torchsymbol` decorator.

Reply:
hmm, maybe no.
I originally tried to make it more graceful in a way but currently it's quite simple as you see.